src/components/ChooseCategoryModal/index.tsx

```diff
@@ -67,7 +80,7 @@ const ChooseCategoryModal = ({ visible, onClose, onSelect }: IModal) => {
           <Image
             style={styles.avatar}
             source={{
-              uri: `https://api.${apiRegion}.amity.co/api/v3/files/${item.avatarFileId}/download`,
+              uri: getAvatarURL(apiRegion, item.avatarFileId),
```

Comment:
i think, it would be better to use sdk function instead. 
e.g. https://github.com/EkoCommunications/AmityUiKitWeb/blob/develop/src/core/hooks/useImage.ts

Reply:
updated krub